README.zh-CN.md

```diff
@@ -61,36 +61,31 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): 使用 AsyncAPI 进行事件模式管理的目录服务。
 - [EventMesh-go](https://github.com/apache/eventmesh-go): EventMesh 运行时的 Go 语言实现。
 
-## 快速入门   
+## 快速入门
 
-本节指南将指导您分别从[本地](#在本地运行-eventmesh-runtime)、[docker](#在-docker-中运行-eventmesh-runtime-)、[k8s](#在-kubernetes-中运行-eventmesh-runtime-)部署EventMesh的步骤:
+本节指南将指导您分别从[本地](#在本地运行-eventmesh-runtime)、[Docker](#在-docker-中运行-eventmesh-runtime)、[K8s](#在-kubernetes-中运行-eventmesh-runtime)部署EventMesh的步骤:
 
-本节指南只是帮助您快速入门EventMesh部署，按照默认配置启动EventMesh，如果您需要更加详细的EventMesh部署步骤，请访问[EventMesh官方文档](https://eventmesh.apache.org/docs/next/introduction)。
+本节指南只是帮助您快速入门 EventMesh 部署，按照默认配置启动 EventMesh，如果您需要更加详细的 EventMesh 部署步骤，请访问[EventMesh官方文档](https://eventmesh.apache.org/docs/next/introduction)。
 
-### 部署 EventMesh Store  
+### 部署 Event Store
 
-> EventMesh现在支持`standalone`、`RocketMQ`、`Kafka`等中间件作为存储   
+> EventMesh 现在支持多个[事件存储](https://eventmesh.apache.org/docs/roadmap#event-store-implementation-status)，默认存储模式为 `standalone`
 > 如果是在非`standalone`模式下，需要先部署所需的`store`，以`rocketmq`模式为例: 部署[RocketMQ](https://rocketmq.apache.org/docs/quickStart/01quickstart/)
```

Comment:
https://github.com/apache/eventmesh/pull/4709#discussion_r1447372653: 

>It will be better to use Event Store instead of `store` to clarify concept.

https://github.com/apache/eventmesh/pull/4709#discussion_r1447379899: 

>I think we can use default configs to launch EventMesh in Quick Start.

```suggestion
> EventMesh 支持[多种事件存储](https://eventmesh.apache.org/docs/roadmap#event-store-implementation-status)，在默认存储模式 `standalone` 下，不依赖其它事件存储作为存储层。
```

Reply:
@Alonexc Please delete "如果是在非`standalone`模式下，需要先部署所需的`store`，以`rocketmq`模式为例" sector.